src/constants.ts

```diff
@@ -18,4 +18,4 @@
  * @public
  */
 
-export const KUBECOST_ANNOTATION_DEPLOYMENT_NAME = 'kubecost.com/deployment-name';
\ No newline at end of file
+export const KUBECOST_DEFAULT_ANNOTATION_DEPLOYMENT_NAME = 'kubecost.com/deployment-name';
```

Comment:
Variable needs to be adapted in this files too:

src/components/useAppData.ts
src/conditions.ts

Reply:
Thanks for pointing that out @phac008, the update has been done.